CHANGELOG.md

```diff
@@ -12,6 +12,12 @@
       * Move `BTPaymentFlowClient+ThreeDSecure` and `BTThreeDSecureRequest` methods to `BTThreeDSecureClient`
       * Remove `BTThreeDSecureError.cannotCastRequest` case
     * Remove dependency on `BraintreePaymentFlow` module
+  * BraintreePaymentFlow
+    * Rename `BTPaymentFlowClient` to `BTLocalPaymentClient`
+    * Rename `BTPaymentFlowResult` to `BTLocalPaymentResult`
```

Comment:
I don't think this helps the merchant that much. `BTLocalPaymentResult` already existed and they never interacted with `BTPaymentFlowResult` on their own.

Reply:
Good call - done: https://github.com/braintree/braintree_ios/pull/1035/commits/98abc978f8d2ac5a869cc8b5d288501d523cd4b9